test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -326,3 +326,59 @@ Describe "help function uses full view by default" -Tags "CI" {
         $gpsHelp | Where-Object {$_ -cmatch '^PARAMETERS'} | Should BeNullOrEmpty
     }
 }
+
+Describe 'help can be found from $home directory' -Tags 'Feature', 'RequireAdminOnWindows', 'RequireSudoOnUnix' {
+    BeforeAll {
+        if([System.Management.Automation.Platform]::IsWindows)
+        {
+            $userHelpRoot = "$HOME/Documents/PowerShell/Help"
+        }
+        else
+        {
+            $userModulesRoot = [System.Management.Automation.Platform]::SelectProductNameForDirectory([System.Management.Automation.Platform+XDG_Type]::USER_MODULES)
+            $userHelpRoot = Join-Path $userModulesRoot -ChildPath ".." -AdditionalChildPath "Help"
+        }
+
+        ## Clear all help from user scope.
+        Remove-Item $userHelpRoot -Force -ErrorAction SilentlyContinue -Recurse
+        Update-Help -Module 'Microsoft.PowerShell.Core' -Scope CurrentUser -Force
+        Update-Help -Module 'Microsoft.PowerShell.Management' -Scope CurrentUser -Force
+        Update-Help -Module 'PSReadLine' -Scope CurrentUser -Force
+        Update-Help -Module 'PackageManagement' -Scope CurrentUser -Force
+
+        ## Delete help from global scope if it exists.
+        $currentCulture = (Get-Culture).Name
+
+        $managementHelpFilePath = Join-Path $PSHOME -ChildPath $currentCulture -AdditionalChildPath 'Microsoft.PowerShell.Commands.Management.dll-Help.xml'
```

Comment:
LInes 352-367 very similar, perhaps change it to a `foreach` loop?

Reply:
Refactored the test file. There is some duplication, but it is much simpler to read now.